benches/lib.rs

```diff
@@ -1,35 +1,38 @@
 use criterion::{criterion_group, criterion_main};
 
 mod to_html {
-    use criterion::Criterion;
+    use criterion::{BenchmarkId, Criterion, Throughput};
     use pulldown_cmark::{html, Options, Parser};
 
-    pub fn benchmarks(c: &mut Criterion) {
-        c.bench_function("pathological_codeblocks1", |b| {
-            // Note that `buf` grows quadratically with number of
-            // iterations. The point here is that the render time shouldn't
-            // grow faster than that.
-            let mut buf = String::new();
-            for i in 1..1000 {
-                buf.push_str(&"`".repeat(i));
-                buf.push(' ');
-            }
-
-            b.iter(|| render_html(&buf, Options::empty()));
-        });
+    pub fn pathological_codeblocks1(c: &mut Criterion) {
```

Comment:
This is my favorite commit. This code now actually directly measures what we're interested in: throughput. Bravo!